src/web/components/Onwards/ExactlyFive.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+import { Card } from '@frontend/web/components/Card/Card';
+import { UL } from '@frontend/web/components/Card/components/UL';
+import { LI } from '@frontend/web/components/Card/components/LI';
+
+type Props = {
+    content: TrailType[];
+};
+
+export const ExactlyFive = ({ content }: Props) => (
+    <>
+        <UL direction="row" bottomMargin={true}>
+            <LI padSides={true} percentage="33%">
+                <Card
+                    {...{
+                        linkTo: content[0].url,
+                        pillar: content[0].pillar,
+                        designType: content[0].designType,
+                        headlineText: content[0].headline,
+                        headlineSize: 'medium',
+                        byline: content[0].byline,
+                        showByline: content[0].showByline,
+                        showQuotes: content[0].designType === 'Comment',
+                        webPublicationDate: content[0].webPublicationDate,
+                        isLiveBlog: content[0].isLiveBlog,
+                        kickerText: content[0].kickerText,
+                        showPulsingDot: content[0].isLiveBlog,
+                        showSlash: true,
+                        showClock: false,
+                        imageUrl: content[0].image,
+                        mediaType: content[0].mediaType,
+                        mediaDuration: content[0].mediaDuration,
+                    }}
+                />
+            </LI>
+            <LI
+                padSides={true}
+                showDivider={true}
+                showTopMarginWhenStacked={true}
+                percentage="33%"
+            >
+                <Card
+                    {...{
+                        linkTo: content[1].url,
```

Comment:
As you repeat this a lot, maybe it could be  a func?

```js
const getCardProps = (content, storyN) => {
  const c = content[storyN];
  return ({
    linkTo: c.url
  })
}
```

Would at least save having to go through each if you want to add/remove a prop. And also make it easier to see the structure of the Components!

Reply:
There are some subtle changes in which properties get called for different cards which could make an abstraction more complicated. Eg. sometimes image isn't included and headlineSize can vary.

If we had exact parity on property names we could get away with spreading the content object and applying any changes afterwards, like:

```
                <Card
                    {...{
                        ...content[0],
                        image: '',
                    }}
                />
```

but we're out of sync on a couple of things so we'd need to be explicit for those

Reply:
Ah ok, I think that'd be a nice enhancement to align those - easier to reason about probably and would make this much cleaner. Future refactor

Reply:
Yes, defo. It's kind of intentionally verbose at the moment with the goal of finding a pattern once we understand the different ways we use it